<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>card</class>
 <widget class="QWidget" name="card">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Task</string>
  </property>
  <widget class="QTextEdit" name="taskName">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>256</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="done">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter your task</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
